src/components/screen/AdminScreen.jsx

```diff
@@ -70,7 +73,10 @@ const AdminScreen = () => {
           icon={<PermIdentity />}
           rows={dummyUsers}
           createContentComponent={<AddUserForm />}
-          //editContentComponent={<AddUserForm />}
+          editContentComponent={
+            <ChangePasswordForm id={rowId ? rowId : undefined} />
+          }
+          setRowId={setRowId}
```

Comment:
Meir clean å gjer det på den her måten https://stackoverflow.com/questions/32370994/how-to-pass-props-to-this-props-children

Reply:
Ok flytter "Id-logikk" ned til *AdminExpansionPanelModal*